From 9b1165724100acabc467ff0699daa8181ad8e881 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 8 Sep 2018 23:00:51 +0300
Subject: [PATCH 1/4] lineage: Enable weather apps

 * Yahoo Weather provider removed as the API was retired [1]

 * Weather Underground API is also being retired [2]
   after Feb 15th 2019 [3]

[1] https://twitter.com/ydn/status/1079784441344446464
[2] https://apicommunity.wunderground.com/weatherapi/topics/end-of-service-for-the-weather-underground-api
[3] https://apicommunity.wunderground.com/weatherapi/topics/weather-underground-api-update

Change-Id: I490c5a7aef90a3cc4749fc681b97c72413a573fb
---
 snippets/lineage.xml | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 3fb2a29..34e031d 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -28,9 +28,7 @@
   <!--<project path="hardware/ti/wlan" name="LineageOS/android_hardware_ti_wlan" />-->
   <!--<project path="hardware/ti/wpan" name="LineageOS/android_hardware_ti_wpan" />-->
   <project path="lineage-sdk" name="LineageOS/android_lineage-sdk" />
-  <!--<project path="lineage-sdk/samples/weatherproviderservice/OpenWeatherMapProvider" name="LineageOS/android_packages_apps_OpenWeatherMapProvider" />-->
-  <!--<project path="lineage-sdk/samples/weatherproviderservice/WundergroundWeatherProvider" name="LineageOS/android_packages_apps_WundergroundWeatherProvider" />-->
-  <!--<project path="lineage-sdk/samples/weatherproviderservice/YahooWeatherProvider" name="LineageOS/android_packages_apps_YahooWeatherProvider" />-->
+  <project path="lineage-sdk/samples/weatherproviderservice/OpenWeatherMapProvider" name="LineageOS/android_packages_apps_OpenWeatherMapProvider" />
   <project path="packages/apps/AppDrawer" name="LineageOS/android_packages_apps_AppDrawer" />
   <project path="packages/apps/AudioFX" name="LineageOS/android_packages_apps_AudioFX" />
   <project path="packages/apps/Eleven" name="LineageOS/android_packages_apps_Eleven" />
@@ -47,7 +45,7 @@
   <project path="packages/apps/Trebuchet" name="LineageOS/android_packages_apps_Trebuchet" />
   <project path="packages/apps/Updater" name="LineageOS/android_packages_apps_Updater" />
   <project path="packages/overlays/Lineage" name="LineageOS/android_packages_overlays_Lineage" />
-  <!--<project path="packages/providers/WeatherProvider" name="LineageOS/android_packages_providers_WeatherProvider" />-->
+  <project path="packages/providers/WeatherProvider" name="LineageOS/android_packages_providers_WeatherProvider" />
   <!--<project path="prebuilts/lineage-sdk" name="LineageOS/android_prebuilts_lineage-sdk" clone-depth="1" />-->
   <project path="system/extras/su" name="LineageOS/android_system_extras_su" />
   <project path="vendor/lineage" name="LineageOS/android_vendor_lineage" />
-- 
2.17.1

